README.md

```diff
@@ -18,10 +18,12 @@ Everytime you try committing, it will first check the quality of the code, then
 
 If you want to compile then use ```sbt update```
 
-To run the pass bakery program, enter this command (Make sure you have your Postgres Server running before running the project):
+To run the pass bakery program, enter this command:
 ```bash
 sbt run
 ```
+**Note**: Everytime you run the program, you need to make sure that the Postgres server is up and running. To do so, Make sure
```

Comment:
https://postgresapp.com/

I've never heard of this, but I suppose this fulfills the requirements. As a note, make sure to specify where to download, and the needed server configurations. No one in our team uses this setup, so you'll have to get familiar with using postgres from the command line with the pgsql cli

Reply:
I did include where to download and how to configure a server in the first paragraph below environment setup list! And yes, I will be familiar with using pgsql cli command line.